test/test_convergence.jl

```diff
@@ -23,77 +23,9 @@ const CP = CLIMAParameters
 const SFP = SF.Parameters
 const APS = CP.AbstractParameterSet
 const TP = Thermodynamics.TemperatureProfiles
-import Thermodynamics.TestedProfiles: input_config, PhaseDryProfiles, PhaseEquilProfiles, shared_profiles
-
+import Thermodynamics.TestedProfiles: input_config, PhaseEquilProfiles
 include(joinpath(pkgdir(SurfaceFluxes), "parameters", "create_parameters.jl"))
-const FT = Float32;
-toml_dict = CP.create_toml_dict(FT; dict_type = "alias")
-param_set = create_parameters(toml_dict, UF.BusingerType())
-thermo_params = SFP.thermodynamics_params(param_set)
-uft = SFP.universal_func_type(param_set)
-
-"""
-    PhaseEquilProfiles(param_set, ::Type{ArrayType})
-
-Returns a `ProfileSet` used to test moist states in thermodynamic equilibrium.
-"""
-function PhaseEquilProfiles(param_set::APS, ::Type{ArrayType}) where {ArrayType}
-    phase_type = TD.PhaseEquil
-
-    # Prescribe z_range, relative_sat, T_surface, T_min
-    z_range, relative_sat, T_surface, T_min = input_config(ArrayType)
-
-    # Compute T, p, from DecayingTemperatureProfile, (reshape RS)
-    z, T_virt, p, RS = shared_profiles(param_set, z_range, relative_sat, T_surface, T_min)
-    T = T_virt
-
-    FT = eltype(T)
-    R_d::FT = CPP.R_d(param_set)
-    grav::FT = CPP.grav(param_set)
-    # Compute total specific humidity from temperature, pressure
-    # and relative saturation, and partition the saturation excess
-    # according to temperature.
-    ρ = p ./ (R_d .* T)
-    q_tot = RS .* TD.q_vap_saturation.(Ref(param_set), T, ρ, Ref(phase_type))
-    q_pt = TD.PhasePartition_equil.(Ref(param_set), T, ρ, q_tot, Ref(phase_type))
 
-    # Extract phase partitioning and update pressure
-    # to be thermodynamically consistent with T, ρ, q_pt
-    q_liq = getproperty.(q_pt, :liq)
-    q_ice = getproperty.(q_pt, :ice)
-    p = TD.air_pressure.(Ref(param_set), T, ρ, q_pt)
-
-    e_int = TD.internal_energy.(Ref(param_set), T, q_pt)
-    θ_liq_ice = TD.liquid_ice_pottemp.(Ref(param_set), T, ρ, q_pt)
-    RH = TD.relative_humidity.(Ref(param_set), T, p, Ref(phase_type), q_pt)
-    e_pot = grav * z
-    Random.seed!(15)
-    u = rand(FT, size(T)) * 5
-    v = rand(FT, size(T)) * 5
-    w = rand(FT, size(T)) * 5
-    e_kin = (u .^ 2 .+ v .^ 2 .+ w .^ 2) / 2
-
-    return ProfileSet{typeof(T), typeof(q_pt), typeof(phase_type)}(
-        z,
-        T,
-        p,
-        RS,
-        e_int,
-        ρ,
-        θ_liq_ice,
-        q_tot,
-        q_liq,
-        q_ice,
-        q_pt,
-        RH,
-        e_pot,
-        u,
-        v,
-        w,
-        e_kin,
-        phase_type,
-    )
-end
 function input_config(ArrayType; n = 10, n_RS1 = 20, n_RS2 = 20, T_surface = 290, T_min = 150)
```

Comment:
@charleskawczynski The purpose of this `input_config` here is to overload the default temperatures in the `TestedProfiles` definition ; can we pass an array of temperatures here?

Reply:
It seems `input_config` is not used in this file?

Reply:
Was this just needed for local experimentation? If so, we can open a PR in Thermodynamics that forwards `input_config_kwargs` from `PhaseEquilProfiles` to `input_config`.

Reply:
For now we are fine using the `input_configs` overload here; we can discuss the changes to Thermodynamics in the next several days.